src/models/discriminant_analysis.jl

```diff
@@ -3,130 +3,123 @@
 ####
 
 @mlj_model mutable struct LDA <: MMI.Probabilistic
-    method::Symbol = :gevd::(_ in (:gevd, :whiten))
-    cov_w::CovarianceEstimator = MS.SimpleCovariance()
-    cov_b::CovarianceEstimator = MS.SimpleCovariance()
-    outdim::Int = 0::(_ ≥ 0)
-    regcoef::Float64 = 1e-6::(_ ≥ 0)
-    dist::SemiMetric = SqEuclidean()
-end
-
-function MMI.fit(model::LDA, ::Int, X, y)
-    Xm_t, yplain, classes_seen, p, n, nc, nclasses, integers_seen, outdim =
-        _check_lda_data(model, X, y)
-    core_res = MS.fit(
-        MS.MulticlassLDA, nc, Xm_t, Int.(yplain);
-        method=model.method,
-        outdim,
-        regcoef=model.regcoef,
-        covestimator_within=model.cov_w,
-        covestimator_between=model.cov_b
-    )
-    cache = nothing
-    report = (
-        classes=classes_seen,
-        outdim=MS.size(core_res)[2],
-        projected_class_means=MS.classmeans(core_res),
-        mean=MS.mean(core_res),
-        class_weights=MS.classweights(core_res),
-        Sw=MS.withclass_scatter(core_res),
-        Sb=MS.betweenclass_scatter(core_res),
-        nclasses=nc
-    )
-    fitresult = (core_res, classes_seen)
-    return fitresult, cache, report
+  method::Symbol = :gevd::(_ in (:gevd, :whiten))
+  cov_w::CovarianceEstimator = MS.SimpleCovariance()
+  cov_b::CovarianceEstimator = MS.SimpleCovariance()
+  outdim::Int = 0::(_ ≥ 0)
+  regcoef::Float64 = 1e-6::(_ ≥ 0)
+  dist::SemiMetric = SqEuclidean()
```

Comment:
Doesn't blue style require 4 spaces for indentation? At least that's what my emacs plugin has always given me. 

```suggestion
    method::Symbol = :gevd::(_ in (:gevd, :whiten))
    cov_w::CovarianceEstimator = MS.SimpleCovariance()
    cov_b::CovarianceEstimator = MS.SimpleCovariance()
    outdim::Int = 0::(_ ≥ 0)
    regcoef::Float64 = 1e-6::(_ ≥ 0)
    dist::SemiMetric = SqEuclidean()
```